speechmatics/cli_parser.py

```diff
@@ -238,6 +238,13 @@ def get_arg_parser():
         default=None,
         help=("Comma-separated list of expected languages for language identification"),
     )
+    config_parser.add_argument(
```

Comment:
config_parser enables the option for both batch and rt, we only want to enable it for batch. Use batch_summarization_parser as example

Reply:
thanks, I changed this